js/widgets/help.js

```diff
@@ -47,6 +47,8 @@ function HelpWidget() {
 
         this._helpDiv.style.width = iconSize * 2 + 425 + "px";
         this._helpDiv.style.backgroundColor = "#e8e8e8";
+        this._helpDiv.style.height = "65%";
```

Comment:
try `maxHeight`. Also, I think 75% or 80% might be better.

Reply:
btw this percentage is relative to the widget window. You need to change the widget window's size, not this context's.

Reply:
>btw this percentage is relative to the widget window. You need to change the widget window's size, not this context's.

thankyou for correcting me :) did the changes.